src/renderer/src/components/bootstrap.tsx

```diff
@@ -39,6 +40,27 @@ const Bootsrap: React.FC = () => {
         });
     });
 
+    // Received when a station's state is updated externally, typically
+    // by another client via a websocket message. When received go through
+    // and ensure the state of the button in TrackAudio matches the new
+    // state in AFV.
+    window.api.on('station-state-update', (data: string) => {
+      const update = JSON.parse(data) as StationStateUpdate;
+
+      useRadioState
+        .getState()
+        .radios.filter((radio) => radio.frequency === update.value.frequency)
```

Comment:
There should only be one, so better that you use "find" and get if you get a nullvalue there

Reply:
Good catch, updated.